### PR TITLE
Posterfix

### PR DESCRIPTION
**Pull request by Pyko1**
:cl: Pyko
fix: Fixed the legit posters
/:cl:

### CODE REVIEW

**Comment by Pyko1** [comment]
might need more fixing if im correct that the map edited serial is also broken.

**Comment by Pyko1** [comment]
Fixed map edited posters also having serial_number = null

**Comment by Pyko1** [comment]
wew also serial/serial_number needs fixing

**Comment by JohnGinnane** [comment]
Did you test this? Check some of the posters around the station ?

**Comment by Pyko1** [comment]
tested on mapped and and rolled posters.

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -259,7 +261,7 @@ list(name = "- Carbon Dioxide", desc = " This informational poster teaches the v
 	var/temp_loc = get_turf(user)
 	flick("poster_being_set",D)
 	D.loc = src
-	D.official = P.official
```
Comment (JohnGinnane) [inline code/game/objects/effects/contraband.dm:262]: I think change this var too to match the weapon object

**Comment by Pyko1** [comment]
even tho i should make it show the icon on mapping as well

File: code/game/objects/effects/contraband.dm (hunk 1 above)
Comment (Pyko1) [inline code/game/objects/effects/contraband.dm]: isnt that changed already?

**Comment by Pyko1** [comment]
even tho too lazy to figure out icons for mapping now so more or less ready

**Pull request merged**